tests/foreman/api/test_subscription.py

```diff
@@ -108,19 +111,14 @@ def test_positive_refresh(request):
 
     :CaseImportance: Critical
     """
-    org = entities.Organization().create()
+    org = function_entitlement_manifest_org
     sub = entities.Subscription(organization=org)
-    with manifests.original_manifest() as manifest:
-        upload_manifest(org.id, manifest.content)
-    request.addfinalizer(lambda: sub.delete_manifest(data={'organization_id': org.id}))
```

Comment:
I think if `delete_manifest` was part of the teardown here then we shouldn't remove this line

And Not sure if a similar `delete_manifest` is required for a teardown for global fixtures?

Reply:
`delete_manifest` was only necessary because of the use of `original_manifest()`: https://github.com/SatelliteQE/robottelo/blob/2e5794e62d381aacde94f01c5265937737165424/robottelo/manifests.py#L183

Each use of `function_entitlement_manifest_org` (or `function_entitlement_manifest`) will create a unique manifest used only for that function. Since the org and the manifest are both function-scoped here, we can safely remove this line.

Reply:
Regarding teardown for global fixtures, deleting the manifest from the organization should not generally be necessary for function- or module-scoped fixtures. However, note that part of the teardown on the Manifester side is to delete the upstream subscription allocation, which will free up the subscriptions in the RHSM account.

Reply:
Sounds great for teardown for global fixtures 👍

and for this particular teardown step, I know it's safe to remove, but I think add/refresh/delete manifest scenarios should be part test_subscription to test api endpoints using `entities.Subscription`, so it would be good to keep this step, right?

Reply:
Fair enough, I hadn't considered that deleting specifically after refreshing should be considered a particular test scenario. I'll add the line back in.